Sensus.UiTest/app.config

```diff
@@ -0,0 +1,15 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <runtime>
+    <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
+      <dependentAssembly>
+        <assemblyIdentity name="Newtonsoft.Json" publicKeyToken="30ad4fe6b2a6aeed" culture="neutral" />
+        <bindingRedirect oldVersion="0.0.0.0-9.0.0.0" newVersion="9.0.0.0" />
+      </dependentAssembly>
+      <dependentAssembly>
+        <assemblyIdentity name="PCLCrypto" publicKeyToken="d4421c8a4786956c" culture="neutral" />
+        <bindingRedirect oldVersion="0.0.0.0-2.0.0.0" newVersion="2.0.0.0" />
+      </dependentAssembly>
+    </assemblyBinding>
+  </runtime>
+</configuration>
```

Comment:
Is this a test? :) You added an app.config file to the Android project so I assume you know?

Lots of things can go into the app.config file. For our purposes though we only seem to use it for backwards compatibility. Because our nuget packages can be updated this file helps make sure breaking updates aren't bound against at runtime. (I think 95% of binding in .net is at runtime. Though now I'm curious about our mobile builds. It must use these bindings at compile time for those environments. Or is Mono installed directly on iOS and Android to run straight CIL?)

These particular backward compatible settings in app.config (along with the file since it didn't exist) were created when updating Nuget packages. I don't know enough about how Nuget packages work to understand why it creates these sometimes while at other times it doesn't.

Does that sound right to you? To be honest, I've never looked into it too much since I've never managed a project with multiple versions of the same assembly.